pkg/private/pkg_files.bzl

```diff
@@ -105,10 +109,12 @@ def _process_pkg_dirs(content_map, pkg_dirs_info, origin, default_mode, default_
             user = attrs[1],
             group = attrs[2],
             origin = origin,
+            uid = attrs[3],
+            gid = attrs[4],
```

Comment:
Consider sorting these differently here and below.  For consistency, it seems that `uid`/`gid` should be below `group` rather than `origin`.

Reply:
done